augur/curate/titlecase.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+Standardizes string fields of the NDJSON record from stdin and outputs the
+modified record to stdout.
```

Comment:
Good catch as that description was because `normalize` and `titlecase` were combined in one script. 

I simplified the description but open to further edits. 
https://github.com/nextstrain/augur/blob/6852345c1976fda35b21583cc256a058bd5f6966/augur/curate/titlecase.py#L2